### PR TITLE
Task/update raja

### PR DESCRIPTION
**Pull request by rhornung67**
This PR updates the RAJA and BLT submodules and updates kernels to conform to RAJA API changes.

### CODE REVIEW

**Review by artv3 — APPROVED** [review]

**Comment by rhornung67** [comment]
@artv3 I will merge after @MrBurmark approves. I want him to look at my changes to the sort kernels.

**Review by MrBurmark — APPROVED** [review]

**Pull request merged**
